addons/hr_contract/models/hr_contract.py

```diff
@@ -16,6 +16,7 @@ class Employee(models.Model):
     medic_exam = fields.Date(string='Medical Examination Date')
     place_of_birth = fields.Char('Place of Birth')
     children = fields.Integer(string='Number of Children')
+    # TODO make a many2one
```

Comment:
Make a bridge. 

Add field vehicle_id to set the employee's vehicle. Override vehicle: related="vehicle_id.name"